src/groovy/au/org/emii/aatams/detection/DetectionView.groovy

```diff
@@ -85,7 +85,7 @@ class DetectionView extends Detection implements Embargoable {
 
     def applyEmbargo(allowSanitised = true) {
 
-        def releaseEmbargoed = surgery.release.isEmbargoed()
+        def releaseEmbargoed = surgery?.release?.isEmbargoed()
```

Comment:
Yeah, I think this could be improved. When I was working on the protected species feature I changed a bunch of code from using the term 'embargo' to 'visibility control' to make it clear that there's more going on now than just embargoes. I obviously didn't go the whole way as there is still and Embargoable interface (which this method belongs to). So I'd agree that the namings here could be improved (and some logic here could be separated into smaller logically parts).

Probably too much to refactor to do it as part of this work.